{ 'init_fall_back': { 'head': [],
                      'left_arm': [],
                      'left_leg': [ { 'duration': 1.0,
                                      'name': u'init_fall_back',
                                      'positions': [ -0.0016,
                                                     0.0006,
                                                     -0.0457,
                                                     -0.0047,
                                                     -0.4961,
                                                     -0.0003],
                                      'starttime': 0.0}],
                      'right_arm': [],
                      'right_leg': [ { 'duration': 1.0,
                                       'name': u'init_fall_back',
                                       'positions': [ 0.0016,
                                                      -0.0006,
                                                      0.0457,
                                                      0.0047,
                                                      0.4961,
                                                      0.0003],
                                       'starttime': 0.0}],
                      'torso': []}}
